### PR TITLE
Add statuscake_ssl_test Ansible module

### PR DESCRIPTION
**Pull request by ronardcaktus**
This PR adds the `statuscake_ssl_test` Ansible module to automate SSL tests using ansible.

**NOTE:** I would suggest a substantial review of this code due to my inexperience with Ansible.

### CODE REVIEW

File: roles/statuscake_monitoring/defaults/main.yml [hunk 1]
```diff
@@ -1,5 +1,14 @@
+# Uptime tests defaults
 statuscake_api_key: ""
 statuscake_default_check_rate: 300
 statuscake_default_test_type: HTTP
 statuscake_uptime_tests: []
 statuscake_merged_uptime_tests: []
+
+# SSL tests defaults
+statuscake_ssl_default_check_rate: 86400
+statuscake_ssl_default_alert_at: [1, 2, 3]
```
Comment (copelco) [inline roles/statuscake_monitoring/defaults/main.yml]: > Comma separated list representing when alerts should be sent (days). Must be exactly 3 numerical values

What does `alert_at` actually mean? Does a value of `10` mean an alert will be sent 10 days before expiration?

**Review by copelco — CHANGES_REQUESTED** [review]
Let's merge `170-SSL-Test` into this branch so we can test invoking the Ansible module.

File: roles/statuscake_monitoring/defaults/main.yml (hunk 1 above)
Comment (ronardcaktus) [reply roles/statuscake_monitoring/defaults/main.yml]: From the documentation:
_"Comma separated list representing when alerts should be sent (days). Must be exactly 3 numerical values"_

I don't like the vagueness, and unfortunately, there is no example.

**Comment by copelco** [comment]
@ronardcaktus Are the refactored role/module changes on another branch? I think we removed the merged variables in the last round of revisions, but I don't see those here.

**Comment by ronardcaktus** [comment]
I believe you mean these changes, @copelco 
https://github.com/caktus/caktus-hosting-services/pull/173

**Comment by copelco** [comment]
Those are the related changes, but this collection also had updates I believe

**Review by copelco — APPROVED** [review]

**Pull request merged**